FWCore/MessageService/interface/MessageServicePresence.h

```diff
@@ -29,7 +28,7 @@ namespace edm {
 
       // --- data:
       edm::propagate_const<std::shared_ptr<ThreadQueue>> m_queue;
-      boost::thread m_scribeThread;
+      std::thread m_scribeThread;
```

Comment:
@Dr15Jones This change is fine, right?

Reply:
I'm not certain we actually use this code.

Reply:
Do you mean the entire `Presence` system (a quick look supports such claim)? 

It is probably easiest to let this change go in, and think of cleaning up what we don't use in the mean time (i.e. opening an issue)